app/models.py

```diff
@@ -0,0 +1,8 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Actor:
```

Comment:
The class name 'Actor' is correctly in singular form, which adheres to the checklist requirement.